phoenix-core-client/src/main/java/org/apache/phoenix/util/ViewUtil.java

```diff
@@ -484,7 +484,8 @@ public static void addIndexesFromParent(PhoenixConnection connection, PTable vie
                             .setTableName(modifiedIndexName)
                             .setViewStatement(viewStatement)
                             .setUpdateCacheFrequency(view.getUpdateCacheFrequency())
-                            .setTenantId(view.getTenantId())
+                            //retain the tenantId from the index being inherited
```

Comment:
@palashc Can you add a testcase in whereoptimizer test with a global view and global view index and a query on tenant view uses index and the scan start row has the tenantid bytes similar to this https://github.com/apache/phoenix/blob/master/phoenix-core/src/test/java/org/apache/phoenix/compile/WhereOptimizerTest.java#L2670-L2671

Reply:
@tkhurana Let me know if the test looks fine - https://github.com/apache/phoenix/pull/1778/commits/d83146b0e66b842774cc024e48aaea290c286cd9

Reply:
Looks good @palashc thanks